pkgs/development/python-modules/aplpy/default.nix

```diff
@@ -0,0 +1,29 @@
+{ pkgs
+, buildPythonPackage
+, fetchPypi
+, numpy
+, astropy
+, matplotlib
+, reproject
+}:
+
+buildPythonPackage rec {
+  pname = "aplpy";
+  version = "2.0rc2";
+
+  doCheck = false; # the tests requires several pytest plugins that are not in nixpkgs
```

Comment:
Can you point me to where you got this info from?

Reply:
From the build log:
```
RuntimeError: Test dependencies are missing. You should install the 'pytest-astropy' package.
builder for '/nix/store/12ni7z8piw98wqjmjjyvcldf20xwzm2g-python3.7-aplpy-2.0rc2.drv' failed with exit code 1
error: build of '/nix/store/12ni7z8piw98wqjmjjyvcldf20xwzm2g-python3.7-aplpy-2.0rc2.drv' failed
```

Reply:
Please mention the concrete package that's missing in the comment.

Reply:
Of course, the optimal thing to do would packaging the dependencies which should not be too difficult.

Reply:
You can use https://github.com/costrouc/nixpkgs-dev-tools to make it easier.

Reply:
I would rather do that in a separate PR.